From: Jan Engelhardt <jengelh@inai.de>
Date: 2018-12-11 03:21:32.396946319 +0100

Put the debug symbols in the libraries so we can have proper
debuginfo packages rather than having a separate debug version
of the libraries.

---
 qcustomplot-sharedlib/sharedlib-compilation/sharedlib-compilation.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: qcustomplot/qcustomplot-sharedlib/sharedlib-compilation/sharedlib-compilation.pro
===================================================================
--- qcustomplot.orig/qcustomplot-sharedlib/sharedlib-compilation/sharedlib-compilation.pro
+++ qcustomplot/qcustomplot-sharedlib/sharedlib-compilation/sharedlib-compilation.pro
@@ -7,7 +7,7 @@ greaterThan(QT_MAJOR_VERSION, 4): QT +=
 
 DEFINES += QCUSTOMPLOT_COMPILE_LIBRARY
 TEMPLATE = lib
-CONFIG += debug_and_release build_all
+CONFIG += release force_debug_info build_all
 static {
   CONFIG += static
 } else {
